tools/cli/src/main.rs

```diff
@@ -38,14 +41,16 @@ pub fn main() -> Result<(), anyhow::Error> {
         .setting(clap::AppSettings::ArgRequiredElseHelp)
         .subcommand(args::build())
         .subcommand(args::embed_wit())
+        .subcommand(args::embed_version())
         .subcommand(args::show_manifest())
         .subcommand(args::show_wit())
         .subcommand(args::repl());
     let arg_matches = app.get_matches();
 
     match arg_matches.subcommand() {
         ("build", Some(args)) => build(args),
-        ("embed", Some(args)) => embed(args),
+        ("embed_it", Some(args)) => embed_wit(args),
```

Comment:
maybe have it as `embed it` & `embed version`? As subcommands to `embed`

Reply:
Discussed in Slack, they become `set version` and `set it`